views/talkpost_do/preview_comment.tt

```diff
@@ -51,6 +51,9 @@ the same terms as Perl itself.  For a copy of the license, please reference
                                 [% END %]
                                 <span class="poster comment-poster">
                                     [% comment.poster %]
+                                    [% IF comment.admin_post %]
+                                        <span>(<span class="admin-post-icon">[% dw.img('admin-post') %]</span> as admin)</span>
```

Comment:
What is the reason for including an additional bare `<span>` here and in the similar block below? Does it serve a function other than syntactic grouping?

Reply:
Great question, I have no idea! But for maintainability I want previews to use the exact same markup as real comments do,\* and [layout dot s2 has spoken](https://github.com/dreamwidth/dw-free/blob/194048b/styles/siteviews/layout.s2#L283-L287) ([myeeeeeah](https://www.youtube.com/watch?v=vNdl9NpCMOA)). 

Poking at the web inspector, it seems to be functionally irrelevant. 🤷 

\* (vague ambition to make the siteviews layout call a .tt template per comment instead of just printing lines so we can have a single path all markup goes through \[same for entry preview\], but that sounds hard so I haven't gotten to it yet.)